Evaluation/Input/evaluation_plan.json

```diff
@@ -180,85 +183,73 @@
             "Project": "Compound",
             "Name": "COMPOUND",
             "Type": "Compound",
-            "SectionId": 31
+            "SectionId": 0
         },
         {
             "Project": "Compound",
             "Name": "Tablet",
             "Type": "Formulation",
-            "SectionId": 31
+            "SectionId": 0
         }
     ],
     "Sections": [
         {
-            "Id": 1,
-            "Title": "1 Introduction",
+            "Reference": "introduction",
+            "Title": "Introduction",
             "Content": "Content/Section1_Introduction.md"
         },
         {
-            "Id": 2,
-            "Title": "2 Methods",
-            "Content": "Content/Section2_Methods.md",
+            "Reference": "methods",
+            "Title": "Methods",
             "Sections": [
                 {
-                    "Id": 21,
-                    "Title": "2.1 Modeling Strategy",
+                    "Reference": "modeling-strategy",
+                    "Title": "Modeling Strategy",
                     "Content": "Content/Section2.1_Modeling_Strategy.md"
                 },
                 {
-                    "Id": 22,
-                    "Title": "2.2 Data",
+                    "Reference": "methods-data",
+                    "Title": "Data",
                     "Content": "Content/Section2.2_Data.md"
                 },
                 {
-                    "Id": 23,
-                    "Title": "2.3 Model Parameters and Assumptions",
+                    "Reference": "model-parameters-and-assumptions",
+                    "Title": "Model Parameters and Assumptions",
                     "Content": "Content/Section2.3_Model_Parameters_and_Assumptions.md"
                 }
             ]
         },
         {
-            "Id": 3,
-            "Title": "3 Results and Discussion",
+            "Reference": "results-and-discussion",
+            "Title": "Results and Discussion",
             "Content": "Content/Section3_Results_and_Discussion.md",
             "Sections": [
                 {
-                    "Id": 31,
-                    "Title": "3.1 Final input parameters",
+                    "Id": 0,
+                    "TODO_REMOVE_ID_AND_KEEP_REF_AFTER_BUGFIX_Reference": "final-input-parameters",
```

Comment:
temporary keeping the Id because of the Bug in RE (a. above) will be replaced as soon as the bug was fixed